templates/zerver/emails/missed_message.txt

```diff
@@ -13,6 +9,15 @@ While you were away you received {{ message_count }} new{% if group_pm %} group
 {% endfor %}
 {% endif %}
 
+—
+Manage email preferences:
+
```

Comment:
These newlines show up in the final email, so should be suppressed (even though I see they were in the original?)